### DISCUSSION TITLE
setValue Rendering issue

### DISCUSSION THREAD

**Discussion by AkshayS-G** [react-hook-form/react-hook-form#10387] [Q&A]
Hi,

In our project we use React hook form, we have 5-6 material ui textfields and we render it in the same DOM, when we console the input it doesn't render, so when we use setValue to register the variable value, the entire set of textfields in the dom re-render.  Also i think my whole component might be rerendering. Iam following all the guidlines formprovider, UseForm, useFormContext, etc. Is there some clues that you guys might provide that can help me prevent re-renders. I would like to know. 

The onchange in the textfield causes the issue cuz one of my fields are dependent on the value of my other textfield..

is there a way to bypass this
Thank you

**Comment by Moshyfawn** [comment]
No way for us to help you without a minimal repro
